src/javascripts/ng-admin/Crud/field/maFileField.js

```diff
@@ -0,0 +1,117 @@
+                            return;
+                        }
+
+                        var uploadParams;
+
+                        scope.files = {};
+                        for (var file in selectedFiles) {
+                            uploadParams = angular.copy(field.config.uploadInformation);
+                            uploadParams.file = selectedFiles[file];
+
+                            $upload
+                                .upload(uploadParams)
+                                .progress(function(evt) {
+                                    scope.files[evt.config.file.name] = {
+                                        "name": evt.config.file.name,
+                                        "progress": Math.min(100, parseInt(100.0 * evt.loaded / evt.total))
+                                    };
+                                })
+                                .success(function(data, status, headers, config) {
+                                    scope.files[config.file.name] = {
+                                        "name": config.file.name,
+                                        "progress": 100
+                                    };
+                                    scope.value = Object.keys(scope.files).join(',');
+                                })
+                                .error(function(data, status, headers, config) {
+                                    delete scope.files[config.file.name];
+
+                                    scope.value = Object.keys(scope.files).join(',');
+                                });
+                        }
+                    };
+
+                    scope.selectFile = function () {
+                        $timeout(function() {
+                            input.click();
+                        }, 0);
+                    };
+                }
+            },
+            template:
+'<div class="row">' +
+    '<div class="col-md-2">' +
+        '<a class="btn btn-default" ng-click="selectFile()">' +
+            '<span>Browse</span>' +
+        '</a>' +
+    '</div>' +
+    '<div class="col-md-10">' +
+        '<div class="row" ng-repeat="file in files track by $index">' +
+            '<div class="col-md-3" style="padding-top: 6px;">' +
+                '<div class="progress" style="margin-bottom: 0;" ng-if="file.progress">' +
+                    '<div class="progress-bar" role="progressbar" aria-valuenow="{{ file.progress }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ file.progress }}%;">' +
+                        '<span class="sr-only">{{ file.progress }}% Complete</span>' +
+                    '</div>' +
+                '</div>' +
+            '</div>' +
+            '<div class="col-md-9" style="padding-top: 6px;"><small><em>{{ file.name }}<em><small></div>' +
+        '</div>' +
+    '</div>' +
+'</div>' +
+'<input type="file" ng-multiple="multiple" accept="accept" ng-file-select ng-model="selectedFiles" ng-file-change="fileSelected($files)"' +
```

Comment:
Because this the `browse` button which is use. It allow overriding default browser upload widget style.